hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -3312,11 +3312,56 @@
         </locks>
     </cp-subsystem>
 
-    <metrics enabled="false" mc-enabled="false" jmx-enabled="false">
-        <collection-interval-seconds>10</collection-interval-seconds>
-        <retention-seconds>30</retention-seconds>
-        <metrics-for-data-structures>true</metrics-for-data-structures>
-        <minimum-level>DEBUG</minimum-level>
+    <!--
+        ===== HAZELCAST METRICS CONFIGURATION =====
+
+        Configuration element's name is <metrics>.
```

Comment:
Don't we need to have `It has the following attributes:` section for `<metrics>` element?

The same question applies to `hazelcast-full-example.yaml`.

Reply:
Good catch, thanks. Added.